analysis_standards.md

```diff
@@ -0,0 +1,51 @@
+# Analysis Standards
+
+The kinds of analyses that the ACF Data Surge Team will engage in can take on a wide range of scopes and levels of complexity. Therefore this guide makes few specific recommendations about exactly what to include in code produced on an analysis project. Instead, this page provides a high-level overview of the components central to a well-executed analysis project:
+
+- [Extraction](#extract)
+- [Initial data validation](#initial-data-validation-and-exploration)
+- [Data cleaning, transformation, and standardization](#data-cleaning-transformation-and-standardization)
+- [Post-cleaning data validation/diagnostics](#post-cleaning-data-validation)
+- [Analysis preparation (optional)](#analysis-preparation)
+- [Analysis](#analysis)
+- [Output](#outputs)
+
+Throughout this page we refer to a module. In Python, a module is essentially any script with the ".py" extension. These scripts can have function, variable, and class definitions which can be imported into other modules. We adopt this Python definition of a module to apply to any language an analysis project is being conducted in. While the text in a given section may refer to a single module, we anticipate that there will definitions imported from elsewhere that are necessary for this single module to run.
+
+## Extract
+This module should extract the data from its raw source and place the data in the input location. If there is no extraction necessary, then make a copy of the raw data and place in the input location. This step prevents corruption of the raw data and lessens the chance that we will have to reach out to our client for a second data delivery due to a mistake on our end.
+
+## Initial data validation and exploration
+This module ensures that all of the variables we expect are on the file, that variables have the type we expect (or we are able to convert to the types desired), that we understand what all of the variables mean, and that we have all of the records expected. These items should be verified assertively. In the event that any of these checks fail, an internal review is necessary to ensure that we are able to recover the information needed for analysis in spite of the failure. At this stage, it is also useful to determine if there are any unusual data patterns. If data are not recoverable or unusual patterns are present in the data, inquire about them with the client.
+
+The initial data validation and exploration step is a good use case for notebooks (i.e. Quarto/Jupyter).
```

Comment:
I think it'd be useful to link "use case for notebooks" to the notebooks standards.